docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,214 @@
+
+```sh
+$ docker build -t my-kube-scheduler:1.0 .
+$ gcloud docker push gcr.io/my-gcp-project/my-kube-scheduler:1.0
+```
+
+### 2. Define a Kubernetes deployment for the scheduler
+
+Like most other system components, Kubernetes allows running a scheduler in a [pod](http://kubernetes.io/docs/user-guide/pods/). Since we have our scheduler in a container image already, we can just create a pod config for it and run it in a Kubernetes cluster. But instead of just running a pod in the cluster, let's use a [Deployment](http://kubernetes.io/docs/user-guide/deployments/) for this example. A [Deployment](http://kubernetes.io/docs/user-guide/deployments/) manages a ReplicaSet which in turn manages the pods, thereby making the scheduler resilient to failures. Here is the deployment config. Save it as `my-scheduler.yaml`:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: my-scheduler
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    matchExpressions:
+    - {key: tier, operator: In, values: [control-plane]}
+    - {key: component, operator: In, values: [scheduler]}
+    - {key: version, operator: In, values: [second]}
+  template:
+    metadata:
+      labels: 
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      containers:
+      - command: [/usr/local/bin/kube-scheduler, --address=0.0.0.0,
+                  --scheduler-name=my-scheduler]
+        image: gcr.io/my-gcp-project/my-kube-scheduler:1.0
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+          initialDelaySeconds: 15
+        name: kube-second-scheduler
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+        resources:
+          requests:
+            cpu: '0.1'
+        securityContext:
+          privileged: false
+        volumeMounts: []
+      hostNetwork: false
+      hostPID: false
+      volumes: []
+```
+
+An important thing to note here is that the name of the scheduler specified as an argument to the scheduler command in the container spec should be unique. This is used to map pods to schedulers.
+
+Please see the [kube-scheduler documentation](http://kubernetes.io/docs/admin/kube-scheduler/) for detailed description of other command line arguments.
+
+### 3. Run the second scheduler in the cluster
+
+In order to run your scheduler in a Kubernetes cluster, just create the deployment specified in the config above in a Kubernetes cluster:
+
+```sh
+$ kubectl create -f my-scheduler.yaml
+```
+
+Verify that the deployment and hence the scheduler pod is running:
+
+```sh
+$ cluster/kubectl.sh get pods --namespace=kube-system
```

Comment:
I always use cluster/kubectl.sh because otherwise I get whatever random version of kubectl shipped with the last Google Cloud SDK version I upgraded to, rather than one that is guaranteed to correspond to the version of the cluster I just kube-up'd. Am I doing something wrong?